test/cli/commands_test.py

```diff
@@ -1601,3 +1601,81 @@ def test_cli_lint_enabled_progress_bar_multiple_files(
         assert r"\rlint by rules:" in raw_output
         assert r"\rrule L001:" in raw_output
         assert r"\rrule L049:" in raw_output
+
+
+multiple_expected_output = """==== finding fixable violations ====
+== [test/fixtures/linter/multiple_sql_errors.sql] FAIL
+L:  12 | P:   1 | L003 | Expected 1 indentations, found 0 [compared to line 10]
+==== fixing violations ====
+1 fixable linting violations found
+Are you sure you wish to attempt to fix these? [Y/n] ...
+Invalid input, please enter 'Y' or 'N'
+Aborting...
+  [4 unfixable linting violations found]
+"""
+
+
+def test__cli__fix_multiple_errors_no_show_errors():
+    """Basic checking of lint functionality."""
+    result = invoke_assert_code(
+        ret_code=1,
+        args=[
+            fix,
+            [
+                "--disable_progress_bar",
+                "test/fixtures/linter/multiple_sql_errors.sql",
+            ],
+        ],
+    )
+    # We should get a readout of what the error was
+    check_a = "4 unfixable linting violations found"
+    assert check_a in result.output
+    # Finally check the WHOLE output to make sure that unexpected newlines are not
+    # added. The replace command just accounts for cross platform testing.
+    assert result.output.replace("\\", "/").startswith(multiple_expected_output)
+
+
+multiple_show_expected_output = """==== finding fixable violations ====
+== [test/fixtures/linter/multiple_sql_errors.sql] FAIL
+L:  12 | P:   1 | L003 | Expected 1 indentations, found 0 [compared to line 10]
```

Comment:
This message has changed:
```
Expected 1 indentation, found 0
```

(Note that "indentation" is singular now.)

We can simply update the test with the new text, but I wonder if there's a way to make the test a bit less fragile if the lint message(s) change again in the future.